### ISSUE TITLE
docker env cannot launch openai.server

### ISSUE THREAD

**Issue by zjc17** [vllm-project/vllm#442]
docker: `nvcr.io/nvidia/pytorch:22.12-py3`
command: `python -m vllm.entrypoints.openai.api_server --host [0.0.0.0](http://0.0.0.0/) --port 8080`
Error:
```
root@ip-172-31-1-200:/workspace# python -m vllm.entrypoints.openai.api_server --host 0.0.0.0 --port 8080
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.8/dist-packages/vllm/entrypoints/openai/api_server.py", line 17, in <module>
    from fastchat.model.model_adapter import get_conversation_template
  File "/usr/local/lib/python3.8/dist-packages/fastchat/model/__init__.py", line 1, in <module>
    from fastchat.model.model_adapter import (
  File "/usr/local/lib/python3.8/dist-packages/fastchat/model/model_adapter.py", line 13, in <module>
    import accelerate
  File "/usr/local/lib/python3.8/dist-packages/accelerate/__init__.py", line 3, in <module>
    from .accelerator import Accelerator
  File "/usr/local/lib/python3.8/dist-packages/accelerate/accelerator.py", line 34, in <module>
    from .checkpointing import load_accelerator_state, load_custom_state, save_accelerator_state, save_custom_state
  File "/usr/local/lib/python3.8/dist-packages/accelerate/checkpointing.py", line 24, in <module>
    from .utils import (
  File "/usr/local/lib/python3.8/dist-packages/accelerate/utils/__init__.py", line 112, in <module>
    from .launch import (
  File "/usr/local/lib/python3.8/dist-packages/accelerate/utils/launch.py", line 27, in <module>
    from ..utils.other import merge_dicts
  File "/usr/local/lib/python3.8/dist-packages/accelerate/utils/other.py", line 24, in <module>
    from .transformer_engine import convert_model
  File "/usr/local/lib/python3.8/dist-packages/accelerate/utils/transformer_engine.py", line 21, in <module>
    import transformer_engine.pytorch as te
  File "/usr/local/lib/python3.8/dist-packages/transformer_engine/__init__.py", line 7, in <module>
    from . import pytorch
  File "/usr/local/lib/python3.8/dist-packages/transformer_engine/pytorch/__init__.py", line 6, in <module>
    from .module import LayerNormLinear
  File "/usr/local/lib/python3.8/dist-packages/transformer_engine/pytorch/module.py", line 16, in <module>
    import transformer_engine_extensions as tex
ImportError: /usr/local/lib/python3.8/dist-packages/transformer_engine_extensions.cpython-38-x86_64-linux-gnu.so: undefined symbol: _ZN3c106detail23torchInternalAssertFailEPKcS2_jS2_RKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
```

**Comment by zjc17** [comment]
Quick reproduction way:
```bash
docker run --gpus all --rm -it nvcr.io/nvidia/pytorch:22.12-py3
pip uninstall pytorch
pip install vllm
python -c 'import transformer_engine_extensions as tex'
```

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: /usr/local/lib/python3.8/dist-packages/transformer_engine_extensions.cpython-38-x86_64-linux-gnu.so: undefined symbol: nvte_layernorm_bwd
```

**Comment by zjc17** [comment]
Solved by downgrade cuda from  12.2 to 11.8
